File: 1832-check-if-the-sentence-is-pangram/1832-check-if-the-sentence-is-pangram.py
class Solution:
    def checkIfPangram(self, sentence: str) -> bool:
        freq=[0]*26
        n=len(sentence)
        for i in range(0,n):
            ch=sentence[i]
            freq[ord(ch)-ord('a')]+=1
        
        #check the frequencies
        m=len(freq)
        for i in range(0,m):
            if(freq[i]==0):
                return False
        return True

File: 1832-check-if-the-sentence-is-pangram/1832-check-if-the-sentence-is-pangram.cpp
class Solution {
public:
    bool checkIfPangram(string sentence) {
        vector<int>freq(26,0);
        int n=sentence.size();
        for(int i=0;i<n;i++){
            int ch=sentence[i];
            freq[ch-'a']++;
        }

        for(int i=0;i<freq.size();i++){
            if (freq[i]==0){
                return false;
            }
        }
        return true;
    }
};